corehq/apps/app_manager/views/apps.py

```diff
@@ -425,6 +425,10 @@ def copy_app(request, domain):
         with zipfile.ZipFile(gzip, 'r', zipfile.ZIP_DEFLATED) as z:
             source = z.read(z.filelist[0].filename)
         app_id_or_source = source
+    elif request.POST.get('build_id'):
+        build = Application.get(request.POST.get('build_id'))
+        build._unset_build_fields()
```

Comment:
those fields get overwritten anyway, this is a safe keeping?

Reply:
They don't get overwritten unless/until someone makes a build.